### PR TITLE
chore(repo): setup

### PR DESCRIPTION
**Pull request by blake-newman**
Create base repository adding the following tooling:

- eslint
- prettier
- rollup
- typescript
- github workflows
- semantic-release (in dry run mode)
- commit lint
- lintstaged + husky
- gitignore
- dependabot

[ch8753]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+.settings
+.settings/*
+.buildpath
+*.iml
+sftp-config*
+*.sublime-*
+
+# Reports #
+######################
+coverage
```
Comment (csi-lk) [inline .gitignore]: CR;LF

File: .lintignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.vscode/
+dist/
+node_modules/
+.pnp.js
+CHANGELOG.md
+packages/cypress-plugin-capture-errors/__tests__/cypress-errors.json
```
Comment (csi-lk) [inline .lintignore]: this needed for this repo?

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (csi-lk) [inline LICENSE.md:1]: Open sourcing this?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Annotations Action
+
+GitHub action for creating annotations from JSON file
+
+## Note
+
+This repository uses npm packages from `@attest` scope on github; we are working hard to open source these packages.
```
Comment (csi-lk) [inline README.md:7]: Can't be open sourced if this is the case?

File: src/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export default function index(): void {}
```
Comment (csi-lk) [inline src/index.ts]: ?

File: README.md (hunk 4 above)
Comment (blake-newman) [reply README.md:7]: It needs to be public for github actions to work

**Review by csi-lk — APPROVED** [review]

**Pull request merged**